lib/cell.rb

```diff
@@ -16,3 +16,16 @@ def fired_upon?
 def fire_upon
 
 end
+pry(main)> cruiser = Ship.new("Cruiser", 3)
+#=> #<Ship:0x00007feb05112d10...>
+
+pry(main)> cruiser.name
+#=> "Cruiser"
+
+pry(main)> cruiser.length
+#=> 3
+
+pry(main)> cruiser.health
+#=> 3
+
+pry(main)> cruiser.sunk?
```

Comment:
We're going to be removing this later on, but happy to see it went through and placed on the appropriate branch. Nice work, Gunnar!